FirmataConstants.h

```diff
@@ -14,268 +14,84 @@
 #ifndef FirmataConstants_h
 #define FirmataConstants_h
 
+namespace firmata {
 /* Version numbers for the Firmata library.
  * The firmware version will not always equal the protocol version going forward.
  * Query using the REPORT_FIRMWARE message.
  */
-#define FIRMATA_FIRMWARE_MAJOR_VERSION  2
-#define FIRMATA_FIRMWARE_MINOR_VERSION  5
-#define FIRMATA_FIRMWARE_BUGFIX_VERSION 4
+static const int FIRMATA_FIRMWARE_MAJOR_VERSION =  2;
```

Comment:
remove FIRMATA_ from this since it's no longer necessary due to the namespace (we'll need to keep it in the define however)